parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -224,6 +233,21 @@ impl NativeTokensMessage {
 	}
 }
 
+pub struct DescribeEthereumAccountKey20Terminal;
+impl DescribeLocation for DescribeEthereumAccountKey20Terminal {
```

Comment:
Maybe a dump question. So what's the benifit  for introduce `DescribeLocation` here?  Seems the internal convert logic still `(b"ethereum", chain_id, key).using_encoded(blake2_256)` not changed?

Reply:
This is just to bring our implementation in line with the new way to convert locations by implementing via the `DescribeLocation` trait. Currently they do not have an implementation in [location_conversion.rs](https://github.com/paritytech/polkadot/blame/master/xcm/xcm-builder/src/location_conversion.rs) for ethereum addresses. We would probably merge our implementation of `DescribeEthereumAccountKey20Terminal` into `xcm-builder` at some point.